server/src/search/search.controller.ts

```diff
@@ -89,49 +89,53 @@ const fullCourseSearch = async ({ search }: SearchQueryType) => {
   return fullSearch;
 };
 
+const courseSlicing = (sorted, searchType: string) => {
+  if (sorted && searchType === "search" && sorted.length > 5) {
+    return sorted.slice(0, 5);
+  }
+
+  if (sorted && searchType === "results" && sorted.length > 200) {
```

Comment:
What are "results" search types being used for if they provide much more information than we can display?